cartography/intel/aws/secretsmanager.py

```diff
@@ -85,12 +182,54 @@ def sync(
     update_tag: int,
     common_job_parameters: Dict,
 ) -> None:
+    """
+    Sync AWS Secrets Manager resources.
+    """
     for region in regions:
         logger.info(
-            "Syncing Secrets Manager for region '%s' in account '%s'.",
-            region,
-            current_aws_account_id,
+            f"Syncing Secrets Manager for region '{region}' in account '{current_aws_account_id}'."
         )
         secrets = get_secret_list(boto3_session, region)
+
+        if not secrets:
+            logger.info(f"No secrets found in region {region}")
+            continue
+
         load_secrets(neo4j_session, secrets, region, current_aws_account_id, update_tag)
+
+        for secret in secrets:
+            logger.info(
+                f"Getting versions for secret {secret.get('Name', 'unnamed')} ({secret['ARN']})"
+            )
+            secret_versions = get_secret_versions(boto3_session, region, secret["ARN"])
+
+            if secret_versions:
```

Comment:
Is it possible to go without this if-else logic? Ideally we don't need to add if-else nesting as much as possible in the sync functions.